modules/fileout_covjson/FoDapCovJsonTransform.cc

```diff
@@ -793,10 +794,11 @@ void FoDapCovJsonTransform::getAttributes(ostream *strm, libdap::AttrTable &attr
             // KENT: The below "if block" is wrong. If the units of lat/lon includes east, north, it may be geographic projection.
             // The ProjectedCRS may imply the 2-D lat/lon. If the variable name is the same as the axis name, and the lat/lon
             // are 1-D, this is a geographic system.
-            if((currUnit.find("east") != string::npos) || (currUnit.find("East") != string::npos) || 
-                (currUnit.find("north") != string::npos) || (currUnit.find("North") != string::npos)) {
-                coordRefType = "ProjectedCRS";
-            }
+            if ((is_geo_dap2_grid == false) && 
+                ((currUnit.find("east") != string::npos) || (currUnit.find("East") != string::npos) || 
```

Comment:
This doesn't work for this case since we are trying to find if the targetValues is part of currUnit. The currUnit is a superset string of the targetValues.